weather_stats/weatherstatsapp/weather/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
I think we should remove the default readme. We have one in the directory above, so feel free to delete this file.